flask/boxwise_flask/graph_ql/resolvers.py

```diff
@@ -69,6 +69,25 @@ def resolve_user(_, info, email):
     return get_user_from_email_with_base_ids(email)
 
 
+@query.field("qrExists")
+def resolve_qr_exists(_, info, qr_code):
+    try:
+        QRCode.get_id_from_code(qr_code)
+    except QRCode.DoesNotExist:
```

Comment:
Maybe it would be semantically cleaner to use or introduce a specific exist or count method on the model which returns false/true?